app/models/concerns/request_helper.rb

```diff
@@ -227,12 +228,11 @@ def set_cookies(uri)
         empty
       end
     end
-  end
 
-  def handle_http_exception_error(error)
-    PenderSentry.notify(e, url: url)
-    Rails.logger.warn level: 'WARN', message: '[Parser] Could not get html', url: url, error_class: e.class, error_message: e.message
-    set_error_callback.call(message: 'Proxy Error', code: Lapis::ErrorCodes::const_get('LOGIN_REQUIRED'))
-    return nil
+    def handle_http_exception_error(error, url)
+      PenderSentry.notify(error, url: url)
+      Rails.logger.warn level: 'WARN', message: '[Parser] Could not get html', url: url, error_class: error.class, error_message: error.message
```

Comment:
Sorry, I overlooked it in my review yesterday too. Also, please update this error message, since this error is not only when getting the HTML. I suggest something like: `[HTTP] HTTP Client Error (Proxy?)`.